test/controllers/integrations/buildkite_controller_test.rb

```diff
@@ -63,4 +63,29 @@
       response.status.must_equal 200
     end
   end
+
+  context 'when the buildkite_release_params hook gets trigger' do
+    before do
+      project.releases.destroy_all
+      project.builds.destroy_all
```

Comment:
can we leave these alone and do a `assert_difference 'project.releases.count', +1 do` ?